content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -151,8 +151,8 @@ kubectl get deployment my-dep                 # List a particular deployment
 kubectl get pods                              # List all pods in the namespace
 kubectl get pod my-pod -o yaml                # Get a pod's YAML
 
-# Describe commands with verbose output
```

Comment:
note: The comment starts with the `kubectl` command string, for example: "Get", "Describe".